crypto.go

```diff
@@ -15,6 +15,7 @@ import (
 	"golang.org/x/crypto/nacl/secretbox"
 )
 
+// EncryptedMessage contains an encrypted message
```

Comment:
Generally I don't think it's worth having docs that state what is already clear from the type. It would be more helpful to link to more information about what the nonce and ciphertext actually are. Don't worry about this now though.

Reply:
I see your point, in this case it is very self-explanatory and the comment is really pedantic. This is more an exercise that I tend to do to be consistent, because the Go linter complains if I don't do it and because it is actually something recommended by the Go style guide here:
https://golang.org/doc/effective_go.html#commentary

_"Inside a package, any comment immediately preceding a top-level declaration serves as a doc comment for that declaration. Every exported (capitalized) name in a program should have a doc comment."_

That said, I can remove if you feel it is superfluous :)